### ISSUE TITLE
False Positive for Open Redirect

### ISSUE THREAD

**Issue by chris-hanna** [security-code-scan/security-code-scan#58]
Are cleansing functions taken into consideration by the analyzer? Not just for open redirect but also xss and other vulnerabilities that need sanitizing.

This still shows a warning:
```
//if (!string.IsNullOrEmpty(model.ReturnUrl))
if (Url.IsLocalUrl(model.ReturnUrl))
{
    return Redirect(model.ReturnUrl);
}
```
The only way to get rid of the warning is to hard code the url string which isn't practical...
return Redirect("www.google.com");

**Comment by JarLob** [comment]
While it should detect it automatically, the other workaround would be to add inline suppression.

**Comment by JarLob** [comment]
Closed via a1a2e92cbfd43dce4c2fb04ef614516eb2c587ed
